aiven_db_migrate/migrate/pgmigrate.py

```diff
@@ -735,6 +747,7 @@ def __init__(
         self.log = logging.getLogger(self.__class__.__name__)
         self.source = PGSource(conn_info=source_conn_info, filtered_db=filtered_db, mangle=mangle)
         self.target = PGTarget(conn_info=target_conn_info, filtered_db=filtered_db, mangle=mangle)
+        self.filtered_db = set(filtered_db.split(",")) if filtered_db else None
```

Comment:
I think this is not used.

Reply:
Thanks. A leftover from the previous version.